src/views/home.hbs

```diff
@@ -1,4 +1,28 @@
-<div> 
-    Hello from home page
+<div id="proposal">
+    <div>
+        Create new post:
+    </div>
+    <textarea onfocus="append()" onblur="remove()" id="text-area" placeholder="Write your proposal..."></textarea>
+    <button id="post">Post</button>
+        <div id="requirements" style="display:none">
+        <ul id="spcialization">
```

Comment:
@Anies12 It looks like your `<ul>` has no `<li>` elements under it, so it might be better to just change this into a div.